bundle/jsky.app.ot/src/main/scala/jsky/app/ot/editor/template/TemplateParametersEditor.scala

```diff
@@ -257,20 +255,26 @@ class TemplateParametersEditor(shells: java.util.List[ISPTemplateParameters]) ex
 
       val JNoneLong: JOption[java.lang.Long] = JNone.instance[java.lang.Long]
 
-      val hms = new HMSFormat()
-      val raField = new BoundTextField[Double](10)(
-        read = s => hms.parse(s),
-        show = hms.format,
-        get  = _.getTarget.getRaHours(JNoneLong).getOrElse(0.0),
-        set  = setTarget((a, b) => a.setRaHours(b))
+      // These are defined in object SPTarget, but private.  Can we open them
+      // up and/or provide get/set for RightAscension/Declination on SPTarget?
+      val ra:  Target @?> RightAscension = Target.coords >=> Coordinates.ra.partial
+      val dec: Target @?> Declination    = Target.coords >=> Coordinates.dec.partial
+
```

Comment:
Yeah I think it makes sense to add them to `TargetLenses` … I was hesitant to add composed lenses since there are potentially a lot of them, but it does make some sense to add anything we end up defining locally more than once.